src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +51,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() >= fromAge
+                        && person.getAge() <= toAge
+                        && person.getSex().equals(Person.Sex.MAN))
```

Comment:
```suggestion
                        && person.getSex() == Person.Sex.MAN)
```
єнами краще порівнювати через ==
https://stackoverflow.com/a/2937561